scenes/game/local_game.gd

```diff
@@ -5014,11 +5364,14 @@ func handle_strike_effect(card_id :int, effect, performing_player : Player):
 			for i in range(MinArenaLocation, MaxArenaLocation + 1):
 				if is_location_in_range(performing_player, attack_card, i):
 					decision_info.limitation.append(i)
-					decision_info.choice.append({
+					var new_choice = {
```

Comment:
was this leftover changes from your previous version?
I guess it's fine to leave in (and above in "place_buddy_into_space", but nothing is using this now right?
maybe also just turn it into an "and"?
Then above in "place_buddy_into_space" you don't actually have to do anything since the and will just process normally. You can remove the if 'additional_effect' code